buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy

```diff
@@ -11,8 +11,12 @@ import java.nio.file.Paths
 
 @CompileStatic
 class IndexGenerator {
-
-    private static final String LATEST_GUIDES_URL = "https://guides.micronaut.io/latest/"
+    private static final String DEFAULT_CARD = "micronauttwittercard.png";
+    private static final String GUIDES_URL = "https://guides.micronaut.io"
+    private static final String LATEST_GUIDES_URL = GUIDES_URL + "/latest/"
+    public static final String TWITTER_MICRONAUT = "@micronautfw"
+    public static final String DEFAULT_TITLE = "Micronaut Guides"
+    public static final String DEFAULT_INTRO = "Step by Step tutorials to learn the Micronaut Framework"
```

Comment:
Both title and intro are mandatory in the guides metadata so those default values would never be used.

Reply:
I prefer to leave them. There will be cases when we need to render index pages, not individual guide pages, and the default card (with default title and intro) will be used for those cases.